examples/sendoauth2.phps

```diff
@@ -0,0 +1,94 @@
+<?php
+
+/**
+ * This example uses the SendOauth2 wrapper to support OAuth2 (and Basic) authentication for both Microsoft
+ * 365 Exchange email and Google Gmail.
+ * Client secrets and X.509 certificates are supported for Exchange. Client secrets are supported for Gmail.
+ * Authorization_code grant flow and client_credentials (i.e. application) grant flow for SMTP are supported for
+ * Exchange. Authorization_code grant flow is supported for Gmail.
+ * Appropriate scopes (client permissions) and 'provider' overrides are added automatically.
+ *
+ * Install with Composer from the decomplexity/SendOauth2 repo.
+ *
+ * SendOauth2 can be also be invoked using less (or even no) arguments - see the repo for details.
+ *
+ * Needs PHPMailer >=6.6.0 that added support for oauthTokenProvider
+ *
+ * (The next release [V4] of the wrapper will replace TheLeague's Google provider by Google's own GoogleOauthClient;
+ * this will provide support for Google's version of client credentials (Service Accounts) and client certificates)
+ */
+
+//Set the wrapper namespace
+namespace decomplexity\SendOauth2;
```

Comment:
This isn't the right way to work with external libraries. The namespace should be up to the developer, and if they want to use your class, they should import it using:

    use decomplexity\SendOauth2\SendOauth2B;

Just as they do for PHPMailer.